playbooks/group_vars/mysql/public.yml

```diff
@@ -45,3 +43,11 @@ consul_service_config:
       - name: "Connect to MySQL"
         tcp: "{{ MYSQL_SERVER_DOMAIN }}:3306"
         interval: "10m"
+
+# FILEBEAT ####################################################################
+
+filebeat_prospectors:
+  - fields:
+      type: "mysql"
+    paths:
+      - /var/log/mysql.*
```

Comment:
It appears that the error logs in that directory are all empty, while the actual logs are in `/var/log/mysql.err`.  I don't fully understand why, though – seems to be some misconfiguration.

Reply:
Yeah, this is catching `/var/log/mysql.err`. There is definitely some misconfiguration.

Reply:
Sorry, I misread this.  And yeah, we can fix the misconfiguration in a different ticket, and it's not urgent.  As long as this doesn't cause any problems, we can ignore it as well.

Reply:
:+1: